tests/utils/useAtomCallback.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import { Provider, useAtom, atom } from '../../src/index'
+import { useAtomCallback } from '../../src/utils'
+
+it('useAtomCallback with get', async () => {
+  const countAtom = atom(0)
+
+  const Parent: React.FC = () => {
+    const [count, setCount] = useAtom(countAtom)
+    const [secondCount, setSecondCount] = useState(0)
+    const readCount = useAtomCallback(
+      useCallback((get) => {
+        const currentCount = get(countAtom)
+
+        setSecondCount(currentCount)
+        return currentCount
+      }, [])
+    )
+    useEffect(() => {
+      const read = async () => await readCount()
+      read()
```

Comment:
```suggestion
      readCount()
```

We could simplify this?